src/Perpetuum/Services/Mail/MailHandler.cs

```diff
@@ -185,5 +185,39 @@ public static int NewMailCount(Character character)
         {
             return Db.Query().CommandText("newMailCount").SetParameter("@characterID", character.Id).ExecuteScalar<int>();
         }
+
+        public static ErrorCodes SendWelcomeMailBeginTutorial(Character newPlayer)
+        {
+            return SendWelcomeMail(newPlayer, 1);
+        }
+
+        public static ErrorCodes SendWelcomeMailExitTutorial(Character newPlayer)
+        {
+            return SendWelcomeMail(newPlayer, 2);
+        }
+
+        private static ErrorCodes SendWelcomeMail(Character newPlayer, int mailId)
```

Comment:
Following my review on OPDB patch for this - change the signature here to pass in the unique name of the pre-made mail message to be sent (think of the name as a value in an enum, and the premadeemail table as a enum table w/ all the associated fields - an enum with benefits)